### DISCUSSION TITLE
Debug a code in django

### DISCUSSION THREAD

**Discussion by Asabi89** [community/community#137297] [Programming Help]
### Select Topic Area

Question

### Body

I need help from a django developer am having issue with the code somebody there to help

**Comment by ebndev** [comment]
Hi @Asabi89, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our `Programming Help` category, as this topic may be more relevant there. It's been a few days so I'm not sure if you're still encountering the problem but in the future, please feel encouraged to post there and provide more details about your issue so someone can give you a nudge in the right direction 😄
